apps/etterlatte-fordeler/src/main/kotlin/behandling/BehandlingKlient.kt

```diff
@@ -25,4 +28,19 @@ class BehandlingKlient(
             setBody(FoedselsNummerMedGraderingDTO(fnr, gradering))
         }.body<ObjectNode>()["id"].longValue()
     }
+
+    suspend fun opprettOppgave(
+        sakId: Long,
+    ): Long {
+        return httpClient.post("$url/oppgaver/sak/$sakId/oppgaver") {
+            contentType(ContentType.Application.Json)
+            setBody(
+                NyOppgaveDto(
+                    OppgaveKilde.EKSTERN,
+                    OppgaveType.MANUELL_JOURNALFOERING,
+                    "Noe feilet. Opprett søknad via Gosys-oppgave",
```

Comment:
Blir ikke det her fryktelig tungvint? Saksbehandler får en oppgave, som forteller at de må opprette en oppgave, for at de skal opprette en behandling? 

Hvorfor ikke bare koble direkte på Gosys og opprette en oppgave? Da sparer man i det minste ett steg

Reply:
Ja, kanskje vi kan det? Versjon 2?